modules/conflict_resolver/php/models/resolveddto.class.inc

```diff
@@ -0,0 +1,67 @@
+<?php declare(strict_types=1);
+/**
+ * PHP version 7
+ *
+ * @category Behavioral
+ * @package  Main
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+namespace LORIS\conflict_resolver\Models;
+
+/**
+ * A ResolvedDTO represents a resolved conflict in the conflict
+ * resolver module. It is doing the mapping between the resolved_conflicts
+ * table columns and the JSON object properties returned to the frontend.
+ *
+ * Additionally, it implements the DataInstance interface so it can be used by
+ * a Database Provisioner.
+ *
+ * @category Behavioral
+ * @package  Main
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+class ResolvedDTO implements \LORIS\Data\DataInstance
+{
+    /**
+     * Implements \LORIS\Data\DataInstance interface for this row.
+     *
+     * @return string
+     */
+    public function toJSON(): string
+    {
+        return json_encode(
+            array(
+                'ResolvedID'          => $this->resolvedid,
+                'Project'             => $this->project,
+                'Site'                => $this->site,
+                'CandID'              => $this->candid,
+                'PSCID'               => $this->pscid,
+                'Visit Label'         => $this->visitlabel,
+                'Instrument'          => $this->instrument,
+                'Question'            => $this->question,
+                'Value 1'             => $this->value1,
+                'Value 2'             => $this->value2,
+                'Correct Answer'      => $this->correctanswer,
+                'User 1'              => $this->user1,
+                'User 2'              => $this->user2,
+                'Resolver'            => $this->resolver,
+                'ResolutionTimestamp' => $this->resolutiontimestamp,
+            )
+        );
+    }
+    /**
+     * Returns the CenterID for this row, for filters such as
+     * \LORIS\Data\Filters\UserSiteMatch to match again.
+     *
+     * @return integer
+     */
+    public function getCenterID(): int
+    {
+        return (int) $this->centerid;
+    }
```

Comment:
get projectId function should be added

Reply:
Why?

Reply:
@ridz1208

Reply:
To filter out other projects data, just like centerid

Reply:
@ridz1208 So what you are asking is for me to add the UserProjectMatch to the data provisioners here : https://github.com/aces/Loris/pull/5140/files#diff-8c4f9a1ba1e40de58508d6300ab817fcR136 and there : https://github.com/aces/Loris/pull/5140/files#diff-401ec2ff7b24354f8856d1462f4eb1f6R148
right?

Reply:
yes please

Reply:
@ridz1208 Done